modules/claims_api/spec/workers/claim_establisher_spec.rb

```diff
@@ -36,9 +36,6 @@
     subject.new.perform(claim.id)
     claim.reload
     expect(claim.evss_id).to eq(1337)
-    expect(claim.form_data).to be_empty
-    expect(claim.auth_headers).to be_empty
-    expect(claim.file_data).to be_nil
```

Comment:
I moved these checks to a better set of specs on the model (see `auto_established_claim_spec.rb`)

Reply:
nice, good additions